resources/views/components/button/conversion.blade.php

```diff
@@ -0,0 +1,12 @@
+{{--
+The enhanced button is a variant specifically used for anything conversion related.
+Like purchasing a product or completing the checkout.
+Example:
+
+```
+<x-rapidez::button.enhanced>Something</x-rapidez::button.enhanced>
+```
+--}}
+<x-rapidez::button {{ $attributes->twMerge('bg-conversion text-conversion-text border-b border-black/15') }}>
```

Comment:
`border-black/15` vs `border` vs `bg-conversion/something`? Or should we introduce `conversion-border` just like `conversion-text`?

Reply:
Removed it here we don't need it as it comes from the `button.blade.php`